core/logger/logger.go

```diff
@@ -189,19 +187,24 @@ func NewLogger() Logger {
 }
 
 type Config struct {
-	LogLevel    zapcore.Level
-	Dir         string
-	JsonConsole bool
-	ToDisk      bool // if false, the Logger will only log to stdout.
-	UnixTS      bool
+	LogLevel                   zapcore.Level
+	Dir                        string
+	JsonConsole                bool
+	UnixTS                     bool
+	DiskMaxSizeBeforeRotate    int // megabytes
+	DiskMaxAgeBeforeDelete     int // days
+	DiskMaxBackupsBeforeDelete int // files
```

Comment:
Where are these numbers set?

Reply:
I've added the logic to load these values. Need confirmation before updating the defaults.